config/locales/en.yml

```diff
@@ -96,6 +96,7 @@ en:
     delete: Delete
     dispute: Dispute
     edit: Edit
+    link: Link
```

Comment:
Can we call this "Public Link" to make it a little clearer what it is? That might mean it no longer makes sense to have this at the top-level `shared`.